components/normalizer/tests/tests.rs

```diff
@@ -1308,6 +1308,19 @@ fn test_utf16_basic() {
     );
 }
 
+#[test]
+fn test_accented_digraph() {
+    let normalizer: DecomposingNormalizer = DecomposingNormalizer::new_nfkd();
+    assert_eq!(
+        normalizer.normalize("\u{01C4}\u{0323}"),
+        "DZ\u{0323}\u{030C}"
+    );
+    assert_eq!(
+        normalizer.normalize("DZ\u{030C}\u{0323}"),
+        "DZ\u{0323}\u{030C}"
+    );
+}
+
```

Comment:
This issue also affects NFD; please add the following test:
```rust
    let normalizer: DecomposingNormalizer = DecomposingNormalizer::new_nfd();
    assert_eq!(normalizer.normalize("\u{0DDD}\u{0334}"), "\u{0DD9}\u{0DCF}\u{0334}\u{0DCA}");
```

Reply:
Done. Thanks. (This character took a lot of attention on the collator side.)